### ISSUE TITLE
Published WCAG2ICT Note SOTD says this is a draft document - which it is not

### ISSUE THREAD

**Issue by maryjom** [w3c/specberus#1957]
For some reason, when the WCAG2ICT Note was published, the SOTD wasn't updated to indicate it was a completed Note. It currently says:
>This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress.

In addition, the document does not contain any information on how to submit comments if there are any issues to report. For example, WCAG 2.2 has the following so I would expect the WCAG2ICT Note to have something similar:
>To comment, [file an issue in the W3C WCAG GitHub repository](https://github.com/w3c/wcag/issues/new). Although the proposed success criteria in this document reference issues tracking discussion, the Working Group requests that public comments be filed as new issues, one issue per discrete comment. It is free to create a GitHub account to file issues. If filing issues in GitHub is not feasible, send email to [public-agwg-comments@w3.org](mailto:public-agwg-comments@w3.org?subject=WCAG%202.2%20public%20comment) ([comment archive](https://lists.w3.org/Archives/Public/public-agwg-comments/)).

Be sure that the next time this is published that the status of the document is reflected correctly.

**Comment by deniak** [comment]
This looks like a bug indeed. Let me get confirmation before fixing the issue.

**Comment by deniak** [comment]
> [@deniak](https://github.com/deniak) Do we have confirmation that this is indeed a bug?
> 
> Thanks.

Not yet, I have a call scheduled tomorrow to discuss that topic.

**Comment by plehegar** [comment]
I agree that this is a bug in pubrules and the requirement for the sentence should be removed.
